### PR TITLE
Allow link to dynamic libpq (.so). Builds OK on ArchLinux.

### PR DESCRIPTION
**Pull request by rechtem**

### CODE REVIEW

File: sources/frontend.c [hunk 1]
```diff
@@ -722,7 +724,7 @@ static od_frontend_status_t od_frontend_remote_client(od_relay_t *relay,
 		server->is_copy = 0;
 		break;
 	case KIWI_FE_QUERY:
-		if (instance->config.log_query)
+		if (instance->config.log_query && route->rule->log_query)
```
Comment (reshke) [inline sources/frontend.c:727]: ```instance->config.log_query || route->rule->log_query```  ?

**Comment by reshke** [comment]
restarted the tests - among the fallen were only flaky ones

File: sources/frontend.c (hunk 1 above)
Comment (mrechte) [reply sources/frontend.c:727]: Hello. I did not intend to include the second commit in this PR. However the second is to give the possibility to log queries on a per route configuration. So in order to to log queries for a given route one would have to set log_query both in the global config and in the route itself.

Comment (mrechte) [reply sources/frontend.c:727]: May be || is better than &&

Comment (reshke) [reply sources/frontend.c:727]: let's change here the condition of disjunction into conjunction and LGTM?

**Review by reshke — APPROVED** [review]
:ok: :cool:

**Comment by mrechte** [comment]
What's wrong with the first commit (cee665f), it seems it has not been taken into account ?

**Comment by x4m** [comment]
@mrechte everything is fine with that commit. It will be squashed during merge.

**Comment by x4m** [comment]
Pushed, thanks Marc!

**Pull request merged**